client/components/AllProducts.js

```diff
@@ -25,7 +19,7 @@ export class AllProducts extends React.Component {
                     <i>
                       <h4>
                         <p>{product.description}</p>
-                        <p>{product.price}</p>
+                        <p>${product.price/100}</p>
```

Comment:
Please wrap this calculation in `Math.floor(...)` so that it doesn't produce results like `$18.99000000000004`

Reply:
And also, needs to be formatted so that it's always 2 decimal places